custom-policies/non-prod/pip-frontend.staging.platform.hmcts.net-B2C_1A_TRUSTFRAMEWORKEXTENSIONS.xml

```diff
@@ -6,72 +6,6 @@
   <BuildingBlocks>

```

Comment:
The localisation and trustframeworkbase files also have references to facebook and social account. Are we not able to remove those?

Reply:
I was originally going to change those, however the recommendation from Azure is not to update the trustframeworkbase files so that any microsoft updates in the future are easier to apply and add any customisation you want into the extensions (or the relying party file itself). Therefore I have left the base file as is, and removed the unneeded bits from the extensions file. The localisation file could technically be updated, but I've kept that as is under a similar principle.